src/helpers/popHelper.ts

```diff
@@ -81,15 +81,15 @@ export function fillPoPTitlesAndAliases(
             const originalTitle = get(masterMeasure, ['measure', 'title']);
```

Comment:
I know it's not new variable but could we use 'master' instead of 'original' in the variable name?

Reply:
Fixed. Is the new name better?

Reply:
ok, cool